config/model.go

```diff
@@ -36,24 +39,102 @@ type ModelArgument struct {
 	FieldWithTypeTag  bool
 }
 
+const (
+	DefaultOutPath = "biz/dal/query"
+	DefaultOutFile = "gen.go"
+)
+
 func NewModelArgument() *ModelArgument {
 	return &ModelArgument{
-		OutPath: "biz/dal/query",
-		OutFile: "gen.go",
+		OutPath: DefaultOutPath,
+		OutFile: DefaultOutFile,
 	}
 }
 
 func (c *ModelArgument) ParseCli(ctx *cli.Context) error {
-	c.DSN = ctx.String(DSN)
-	c.Type = strings.ToLower(ctx.String(DBType))
-	c.Tables = ctx.StringSlice(Tables)
-	c.OnlyModel = ctx.Bool(OnlyModel)
-	c.OutPath = ctx.String(OutDir)
-	c.OutFile = ctx.String(OutFile)
-	c.WithUnitTest = ctx.Bool(UnitTest)
-	c.ModelPkgName = ctx.String(ModelPkgName)
-	c.FieldNullable = ctx.Bool(Nullable)
-	c.FieldWithIndexTag = ctx.Bool(IndexTag)
-	c.FieldWithTypeTag = ctx.Bool(TypeTag)
+	configPath := ctx.String(C)
+	dsn := ctx.String(DSN)
+	tp := strings.ToLower(ctx.String(DBType))
+	tables := ctx.StringSlice(Tables)
+	onlyModel := ctx.Bool(OnlyModel)
+	outPath := ctx.String(OutDir)
+	outFile := ctx.String(OutFile)
+	withUnitTest := ctx.Bool(UnitTest)
+	modelPkgName := ctx.String(ModelPkgName)
+	fieldNullable := ctx.Bool(Nullable)
+	fieldWithIndexTag := ctx.Bool(IndexTag)
+	fieldWithTypeTag := ctx.Bool(TypeTag)
+	if configPath != "" {
```

Comment:
How is the priority of this configuration file determined? 
Maybe add some description  both in your pr and in the code comment.

Reply:
sure, 
priority:
command line > config file